deploy/template.yaml

```diff
@@ -362,8 +362,6 @@ Resources:
               Value: !Sub
                 - "cri-passport-front-sessions-${Environment}"
                 - Environment: !Ref Environment
-            - Name: EXTERNAL_WEBSITE_HOST
```

Comment:
Why is this being removed?

Reply:
I don't recall, it was removed on a pair programming call with you.